main.go

```diff
@@ -444,8 +444,55 @@ func hashWith(salt, value string) string {
 	return "z" + sum[:length]
 }
 
+type blacklistItem struct {
+	pkg      string
+	typename string
+}
+
+func buildBlacklist(files []*ast.File, info *types.Info) []blacklistItem {
+	var (
+		blacklist   []blacklistItem
+		prevReflect bool
+	)
+
+	pre := func(cursor *astutil.Cursor) bool {
+		node, ok := cursor.Node().(*ast.Ident)
```

Comment:
you don't want to search for identifiers. you want to search for `*ast.CallExpr`, then you don't need to do the prevReflect dance (which I'm pretty sure is buggy)